### ISSUE TITLE
condor_history reports exit code incorrectly

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/saga-python#582]
Run this job:

```
+ProjectName = "TG-CCR140028"
universe     = vanilla
executable   = /bin/false
request_cpus = 1
queue
```

and query state after completion:

```
$ condor_history 31450833 -autoformat:v, ProcId JobStatus ExitStatus CompletionDate
0, 4, 0, 1471518210
```

where SAGA interprets the `JobStatus` value of `4` as `DONE`, which seems the correct interpretation wrt. condor documentation.  I'll open a condor ticket for this one.  Meanwhile, we cannot rely upon the final state to be correct.

**Comment by andre-merzky** [comment]
```
ExitStatus:
  The way that Condor previously dealt with a job's exit status. This attribute should no longer be used. It is not always accurate in heterogeneous pools, or if the job exited with a signal. Instead, see the attributes: ExitBySignal, ExitCode, and ExitSignal.
```

**Comment by andre-merzky** [comment]
This has been fixed in 7b2b3e6
